tools/accuracy_checker/configs/bert-large-uncased-whole-word-masking-squad-fp32-onnx-0001.yml

```diff
@@ -0,0 +1,38 @@
+models:
+  - name: bert-large-uncased-whole-word-masking-squad-fp32-onnx-0001
```

Comment:
The name here doesn't match the name of the model folder. I believe that the name here is what's used in our internal infrastructure, so please adjust the folder name and the doc to match.

Reply:
Done

Reply:
Done

Reply:
What did you do? The names still don't match.

Reply:
Sorry about that, hopefully this is fixed now. It is getting increasingly hard to keep track of OMZ's naming conventions.

Reply:
Well... the names do match now, but they aren't the correct names. I told you, the name that was in this file was the correct one, because that's the one we're using internally.

Reply:
I pushed one more attempt, but if the naming is still incorrect, then could you please modify the PR yourself as necessary in order to make it right? It seems that deducing a way of correctly setting up all N variants of the same model name in all M required places - at least in a manner that is consistent with the naming of other models - is beyond my abilities.